package.json

```
@@ -28,6 +32,6 @@
     "watchify": "^3.11.0"
   },
   "dependencies": {
-    "@mparticle/web-sdk": "^2.12.2"
+    "@mparticle/web-sdk": "^2.20.2"
```

latest is 2.20.0,  2.20.2 doesn't exist yet 😁

```suggestion
    "@mparticle/web-sdk": "^2.20.0"
```